pallets/ocex/src/lib.rs

```diff
@@ -304,19 +304,50 @@ pub mod pallet {
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		/// On idle, use the remaining weight to do clean up, remove all ingress messages that are
 		/// older than the block in the last accepted snapshot.
```

Comment:
Is this still valid doc?

Reply:
No, it should be removed. We don't remove ingress messages anymore

Reply:
Updated